examples/gocean/eg3/compute_cu_mod.f90

```diff
@@ -63,7 +63,7 @@ module compute_cu_mod
      !! point. This kernel assumes that the U,V and F points that
      !! share the same index as a given T point are those immediately
      !! to the South and West of it.
-     integer :: index_offset = GO_OFFSET_SW
```

Comment:
As already said, we can't change this without changing the kernel implementation.

Reply:
Reverted the changes and added a TODO to the relevant dl_esm_inf issue.